src/App.js

```diff
@@ -1,25 +1,74 @@
-import logo from './logo.svg';
 import './App.css';
+import axios from 'axios';
+import { Component } from 'react';
+import Figure from 'react'; 
+import Container from 'react';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: '',
+      location: {},
+      error: false,
+    }
+  }
+  handleChange = event => {
+    this.setState({
+      searchQuery: event.target.value,
+    });
+  }
+
+  getLocation = async () => {
+
+    const url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&Q=${this.state.searchQuery}&format=json`;
+
+    try {
+      const response = await axios.get(url);
+
+      const location = response.data[0];
+
+      this.setState({
+        location: location,
+        error: false,
+      });
+
+    } catch (error) {
+      console.error('Invalid City', this.state.searchQuery);
+      this.setState ({ error: true});
+    }
+  }
+
+  render() {
+    return (
+      <Container>
+        <input onChange={(event) => this.setState({ searchQuery: event.target.value})} placeholder="enter city name"></input>
```

Comment:
Nice work gettin this going! 👍 

Feedback here: remove setState call here in render. This can cause some funky things to happen (infinite loops, and also make the component less maintainable). In general this is something most folks try and steer away from. 

Instead create a method to handle the event outside of render and pass it to onChange. Get with a TA to help you if you have questions 👍